hidden/Common/Util.swift

```diff
@@ -29,5 +29,18 @@ class Util {
         let prefWindow = PreferencesWindowController.shared.window
         prefWindow?.bringToFront()
     }
+    
+    static var menuBarIsInUse : Bool {
```

Comment:
This break function of app in my machine. There are no menu opened. It prevented app working. I try to return false and anything working well.

Reply:
Btw I don't see any Grant access popup or do anything and detect mouse on statusbar still working.
Accessibility access not granted
Here is the window make function return menu in use
(lldb) po info
▿ 10 elements
  ▿ 0 : 2 elements
    - key : "kCGWindowOwnerName"
    - value : axAuditService
  ▿ 1 : 2 elements
    - key : "kCGWindowOwnerPID"
    - value : 5592
  ▿ 2 : 2 elements
    - key : "kCGWindowAlpha"
    - value : 1
  ▿ 3 : 2 elements
    - key : "kCGWindowMemoryUsage"
    - value : 1232
  ▿ 4 : 2 elements
    - key : "kCGWindowNumber"
    - value : 39076
  ▿ 5 : 2 elements
    - key : "kCGWindowSharingState"
    - value : 0
  ▿ 6 : 2 elements
    - key : "kCGWindowBounds"
    ▿ value : 4 elements
      ▿ 0 : 2 elements
        - key : X
        - value : 0
      ▿ 1 : 2 elements
        - key : Height
        - value : 900
      ▿ 2 : 2 elements
        - key : Y
        - value : 0
      ▿ 3 : 2 elements
        - key : Width
        - value : 1440
  ▿ 7 : 2 elements
    - key : "kCGWindowIsOnscreen"
    - value : 1
  ▿ 8 : 2 elements
    - key : "kCGWindowStoreType"
    - value : 1
  ▿ 9 : 2 elements
    - key : "kCGWindowLayer"
    - value : 2998

Reply:
It looks like `axAuditService` has an invisible window covering the entire screen, which has `kCGWindowLayer` = 2998. I might have to add more restrictions to the detection of menu bar.

Reply:
I believe this is solved in the latest commit